### PR TITLE
feat: integrating plugins and scripts

### LINKED ISSUE #70: Refactoring plugin.js and scripts.js as a whole
## Is your feature request related to a problem? Please describe.

These files use similar processes.
So, refactoring will help development.

## Describe the solution you'd like

Refactor similar processes of `plugin.js` and `scripts.js`.

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Remove code related to the script. This will require integration of `plugins_list.xml` with `scripts_list.xml`.

BREAKING CHANGE
- Data about the scripts is deleted.
- Change the file name from `plugins_list.xml` to `packages_list.xml`. The new tag names are `<packages>` and `<package>`.

## Related Issue

fix #70

## How Has This Been Tested?

- Using the ~~`plugins_list.xml`~~`packages_list.xml`  attached below, you can install and uninstall the plugin and script without any problems.

~~[plugins_list.zip](https://github.com/hal-shu-sato/apm/files/7110627/plugins_list.zip)~~
[packages_list.zip](https://github.com/hal-shu-sato/apm/files/7111049/packages_list.zip)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by hal-shu-sato** [comment]
In relation to this PR, I would like to call the plugin and the script together as a package, what do you think?

**Comment by mitosagi** [comment]
I think calling it a package in the code or xml is a good idea.

The exception is that I think many people who don't do programming are not familiar with calling a zip file a package, so I'd like to call it plugin&script on the UI.

Do you think this is a good way to go?

**Comment by hal-shu-sato** [comment]
Yes, I think so too.

Also, after all, apm is dealing with scripts as well as plugins, so I'm starting to think that the name of the application should be AviUtl Package Manager. (As in #90)
What do you think about it?

**Comment by mitosagi** [comment]
I prefer the "plugin manager", which is easier to understand the features. But the "package manager" is also good.

**Comment by hal-shu-sato** [comment]
I see...
But I'm afraid that "plugin" means that apm only deals with plugins, not scripts.
That bothers me.

**Comment by mitosagi** [comment]
I replaced "plugin" with "package" in the code. I've changed the XML tag names, so I' ll send a PR to apm-data as well after the merge.

**Comment by mitosagi** [comment]
Certainly, it would be better to use a name that covers the function, so that we won't regret it later.

**Comment by hal-shu-sato** [comment]
Certainly...
I'll send an issue later.

File: src/package/package.js [hunk 1]
```diff
@@ -168,6 +186,7 @@ module.exports = {
           .filter((i) => i.isFile() && regex.test(i.name))
           .map((i) => i.name);
       return readdir(instPath).concat(
+        readdir(path.join(instPath, 'packages')).map((i) => 'packages/' + i),
```
Comment (hal-shu-sato) [inline src/package/package.js:189]: There is a bug.
I don't have enough time to suggest all the fixes, so I apologize for the messy suggestions.

```suggestion
        readdir(path.join(instPath, 'plugins')).map((i) => 'plugins/' + i),
```

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
There is a bug.
Please check it.
But thank you for making this massive change.

File: src/package/package.js (hunk 1 above)
Comment (mitosagi) [reply src/package/package.js:189]: Oh, I certainly didn't check there.

I checked the other strings, but I think the only part I need to change is where you suggested.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for your commit!!

**Pull request merged**